### PR TITLE
libbpf-tools: fix some tools error info and usage

### PR DESCRIPTION
**Pull request by ethercflow**
- Fix the error message in the open phase
- Fix spelling error: ojbect -> object
- Complete parameter information in usage
- Capitalize parameters that do not require parameter values ​​for easy memory

Signed-off-by: Wenbo Zhang <ethercflow@gmail.com>

### CODE REVIEW

File: libbpf-tools/biolatency.c [hunk 1]
```diff
@@ -40,20 +40,20 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
...
 "Summarize block device I/O latency as a histogram.\n"
 "\n"
-"USAGE: biolatency [--help] [-T] [-m] [-Q] [-D] [-F] [-d] [interval] [count]\n"
+"USAGE: biolatency [--help] [-T] [-M] [-Q] [-D] [-F] [-d DISK] [interval] [count]\n"
 "\n"
 "EXAMPLES:\n"
 "    biolatency              # summarize block I/O latency as a histogram\n"
 "    biolatency 1 10         # print 1 second summaries, 10 times\n"
-"    biolatency -mT 1        # 1s summaries, milliseconds, and timestamps\n"
+"    biolatency -MT 1        # 1s summaries, milliseconds, and timestamps\n"
```
Comment (anakryiko) [inline libbpf-tools/biolatency.c:48]: biolatency.py is using `-m`, why sudden and diverging change?

File: libbpf-tools/cpudist.c [hunk 2]
```diff
@@ -37,20 +37,20 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
...
 "\n"
-"USAGE: cpudist [--help] [-O] [-T] [-m] [-P] [-L] [-p PID] [interval] [count]\n"
+"USAGE: cpudist [--help] [-O] [-T] [-M] [-P] [-L] [-p PID] [interval] [count]\n"
 "\n"
 "EXAMPLES:\n"
 "    cpudist              # summarize on-CPU time as a histogram"
 "    cpudist -O           # summarize off-CPU time as a histogram"
 "    cpudist 1 10         # print 1 second summaries, 10 times"
-"    cpudist -mT 1        # 1s summaries, milliseconds, and timestamps"
+"    cpudist -MT 1        # 1s summaries, milliseconds, and timestamps"
```
Comment (anakryiko) [inline libbpf-tools/cpudist.c:46]: similarly. py version is sticking with -m. What's your rationale for making these incompatible changes?

File: libbpf-tools/drsnoop.c [hunk 3]
```diff
@@ -31,18 +31,18 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
...
 "    drsnoop -p 123  # trace pid 123\n"
 "    drsnoop -t 123  # trace tid 123 (use for threads only)\n"
 "    drsnoop -d 10   # trace for 10 seconds only\n"
-"    drsnoop -e      # trace all direct reclaim events with extended faileds\n";
+"    drsnoop -E      # trace all direct reclaim events with extended faileds\n";
 
 static const struct argp_option opts[] = {
 	{ "duration", 'd', "DURATION", 0, "Total duration of trace in seconds" },
-	{ "extended", 'e', NULL, 0, "Extended fields output" },
+	{ "extended", 'E', NULL, 0, "Extended fields output" },
```
Comment (anakryiko) [inline libbpf-tools/drsnoop.c:45]: this seems to be libbpf-tools exclusive argument, but again, why changing it in the first place?

**Review by anakryiko — CHANGES_REQUESTED** [review]
There are a bunch more cases of arbitrary changing the case of existing command line arguments with no justification. Please provide rationale for the changes in your commit messages, not just the "what".

Also, while I have your attention. Can you please check https://github.com/iovisor/bcc/issues/3224#issuecomment-759909082? Would you be able to add that check? Thanks!

File: libbpf-tools/biolatency.c (hunk 1 above)
Comment (ethercflow) [reply libbpf-tools/biolatency.c:48]: > There are a bunch more cases of arbitrary changing the case of existing command line arguments with no justification. Please provide rationale for the changes in your commit messages, not just the "what".

Oh, my motivation is I think maybe it's easier to remember. Lowercase parameters need parameter values ​​(except -v), and uppercase ones belong to the switch class, so you don't need to add parameter values.

> Also, while I have your attention. Can you please check [#3224 (comment)](https://github.com/iovisor/bcc/issues/3224#issuecomment-759909082)? Would you be able to add that check? Thanks!

Oh sorry, I haven't checked the email from iovisor/bcc recently, I missed it. I will look at it later （Maybe weekend）.

Comment (anakryiko) [reply libbpf-tools/biolatency.c:48]: I haven't heard about such naming convention, tbh. I think the harm of changing existing arguments and, especially, diverging from .py tool, is much bigger. So let's not do such renamings unnecessarily.

And thanks for checking the other PR.

Comment (ethercflow) [reply libbpf-tools/biolatency.c:48]: Got it,  I will be consistent with .py tool. Thanks!

**Pull request closed without merging**